src/App.jsx

```diff
@@ -1,104 +1,19 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+export const posts = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(item => item.id === post.userId),
+  comments: commentsFromServer.filter(item => item.postId === post.id),
```

Comment:
- Item is a bad name, variable/arguments names should describe them
- You can create separate helper functions to do this operations